eli5/xgboost.py

```diff
@@ -91,7 +93,9 @@ def explain_prediction_xgboost(
         target_names=None,
         targets=None,
         feature_names=None,
-        vectorized=False):
+        vectorized=False,
+        expand_missing_features=False,
```

Comment:
A more general implementation could be the following: 

1. support feature_re for explain_prediction;
2. add an option to show combined weights for filtered out features.

But `collapse_missing_features` also sounds fine to me; it is already implemented, and it is a bit different from feature_re if there are non-missing features with `(missing)` in their names.